.github/kokoro/steps/hostsetup.sh

```diff
@@ -58,6 +59,12 @@ sudo apt-get install -y \
         #Don't include libtbb-dev since it may increase memory usage
         #libtbb-dev \
 
+export PATH="$PATH:/home/kbuilder/.local/bin"
+curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+python3 get-pip.py   
+rm get-pip.py
+python3 -m pip install -r ../../../requirements.txt
```

Comment:
I believe this should simply be "requirements.txt", as the working directory isn't actually changed to this folder.